python_modules/dagster-pipes/dagster_pipes/__init__.py

```diff
@@ -197,19 +197,28 @@ def _resolve_optionally_passed_asset_key(
     asset_key: Optional[str],
     method: str,
 ) -> str:
-    asset_keys = _assert_defined_asset_property(data["asset_keys"], method)
```

Comment:
this was failing since we don't populate `asset_keys` on `@asset_check` invocation 

To save time on this detour i didn't dig in on if it would make sense to populate asset_keys (or check_name ?) in to the context to facilitate not having to pass those values

Reply:
> To save time on this detour i didn't dig in on if it would make sense to populate asset_keys (or check_name ?) in to the context to facilitate not having to pass those values

For sure. We should provide these automatically when it makes sense to.